pom.xml

```diff
@@ -62,7 +62,6 @@
 	</developers>
 
 	<modules>
-		<module>example</module>
```

Comment:
removed `example` from the modules so that it's obvious that you need to run `mvn package` inside the `example` folder separately